cmd/snap/cmd_known.go

```diff
@@ -31,11 +31,25 @@ import (
 	"github.com/jessevdk/go-flags"
 )
 
+type assertTypeName string
+
+func (n assertTypeName) Complete(match string) []flags.Completion {
+	names := asserts.TypeNames()
+	ret := make([]flags.Completion, 0, len(names))
+	for _, name := range names {
+		if strings.HasPrefix(name, match) {
+			ret = append(ret, flags.Completion{Item: name})
```

Comment:
we can! it's not used for much though. Zsh could make use of them, but I don't know of anybody actually doing the work for that to happen. If you have `zsh`, one program that uses this in its completion is `figlet`:

     % figlet -␉
    Completing option
    -C  -- specify control file
    -D  -- use Deutsch character set
    -E  -- use English character set
    -I  -- display info
    …